stable/airflow/values.yaml

```diff
@@ -17,40 +17,20 @@ airflow:
   #   subPath: file.yaml
 
 
-  ## When existingAirflowSecret is defined, secretsMapping can be
-  ## overridden. When no secretName is given then the value of
-  ## existingAirflowSecret is assumed.
-  ## secretsMapping:
-  ## - envVar: AIRFLOW__LDAP__BIND_PASSWORD
-  ##   secretName: ldap
-  ##   secretKey: ldapBindPassword
-  ## - envVar: AIRFLOW__ATLAS__PASSWORD
-  ##   secretKey: atlasPassword
-  ## - envVar: AIRFLOW__SMTP__PASSWORD
-  ##   secretKey: smtpPassword
-  ## - envVar: AIRFLOW__KUBERNETES__GIT_PASSWORD
-  ##   secretKey: kubernetesGitPassword
-  ## - envVar: POSTGRES_USER
-  ##   secretName: postgres
-  ##   secretKey: postgresUser
-  ## - envVar: POSTGRES_PASSWORD
-  ##   secretName: postgres
-  ##   secretKey: postgresPassword
-  ## - envVar: REDIS_PASSWORD
-  ##   secretName: redis
-  ##   secretKey: redisPassword
-  secretsMapping:
-
+  ##
+  ## Extra environment variables to mount in the web, scheduler, and worker pods:
```

Comment:
This new syntax support setting environment variables using the full Kubernetes syntax, including secrets from environment variables and even variables derived from other environment variables.

Reply:
Is it like this the other chart work? The only drawback I can see is that passwords and creds are not stored in a secret anymore. I don’t think it is very important